docs/UserGuide.md

```diff
@@ -180,7 +182,7 @@ Examples:
 - `filter c/Transport`
 - `filter ft/2022-03 c/Transport`
 
-_This is the result of executing the command(s) above:_
+_This is the result of executing the command `filter ft/2022-03`:_
```

Comment:
May need to place in disclaimer say what is in the sample data before executing this command. In addition, may need to disclaim the results may vary depending on the data you input. At current, although obvious, but it could be misleading.

Reply:
done